From 9452bc1864d7a40727e4be0f9e48470d40a31fea Mon Sep 17 00:00:00 2001
From: Xianzhong <b07117@freescale.com>
Date: Wed, 16 Jan 2013 02:04:52 +0800
Subject: [PATCH 149/304] ENGR00239939 Change filter stretch property name

The property name is changed in gpu hwcomposer by vendor.

Signed-off-by: Xianzhong <b07117@freescale.com>
---
 hdmidongle_6dq/init.rc | 2 +-
 sabreauto_6q/init.rc   | 2 +-
 sabresd_6dq/init.rc    | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/hdmidongle_6dq/init.rc b/hdmidongle_6dq/init.rc
index 847e0b6..bea3852 100755
--- a/hdmidongle_6dq/init.rc
+++ b/hdmidongle_6dq/init.rc
@@ -30,7 +30,7 @@ on boot
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
 
-    setprop hwc.filter_stretch  1
+    setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
 
     # fsl omx graphic manager media framework property
diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index b0a854f..172a293 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -30,7 +30,7 @@ on boot
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
 
-    setprop hwc.filter_stretch  1
+    setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
 
     # fsl omx graphic manager media framework property
diff --git a/sabresd_6dq/init.rc b/sabresd_6dq/init.rc
index f49ff2c..769118c 100755
--- a/sabresd_6dq/init.rc
+++ b/sabresd_6dq/init.rc
@@ -38,7 +38,7 @@ on boot
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
 
-    setprop hwc.filter_stretch  1
+    setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
 
     # fsl omx graphic manager media framework property
-- 
1.8.0

